robot_map/Object[] objects
